### PR TITLE
Add preference for single quit button mode

### PR DESCRIPTION
**Pull request by natewalck**
Tested both modes, works! :)

### CODE REVIEW

File: Nudge/Preferences/PreferencesStructure.swift [hunk 1]
```diff
@@ -296,6 +296,7 @@ struct UserExperience: Codable {
     var noTimers: Bool?
     var nudgeRefreshCycle: Int?
     var randomDelay: Bool?
+    var singleQuitButton: Bool?
```
Comment (erikng) [inline Nudge/Preferences/PreferencesStructure.swift:299]: I'm surprised this worked. There is a another place in this file that this key needs to be set. do a search for the key above and you will see where.

**Review by erikng — CHANGES_REQUESTED** [review]

File: Nudge/Preferences/DefaultPreferencesNudge.swift [hunk 2]
```diff
@@ -37,6 +37,7 @@ let maxRandomDelayInSeconds = userExperienceProfile?["maxRandomDelayInSeconds"]
 let noTimers = userExperienceProfile?["noTimers"] as? Bool ?? userExperienceJSON?.noTimers ?? false
 let nudgeRefreshCycle = userExperienceProfile?["nudgeRefreshCycle"] as? Int ?? userExperienceJSON?.nudgeRefreshCycle ?? 60
 let randomDelay = userExperienceProfile?["randomDelay"] as? Bool ?? userExperienceJSON?.randomDelay ?? false
+let singleQuitButton = userExperienceProfile?["singleQuitButton"] as? Bool ?? userExperienceJSON?.singleQuitButton ?? false
```
Comment (erikng) [inline Nudge/Preferences/DefaultPreferencesNudge.swift:40]: this should be in `userInterface` since that is where simpleMode is and all of the button text

**Review by erikng — CHANGES_REQUESTED** [review]

File: Nudge/Preferences/PreferencesStructure.swift (hunk 1 above)
Comment (natewalck) [reply Nudge/Preferences/PreferencesStructure.swift:299]: I think the latest push gets this done.

File: Nudge/Preferences/DefaultPreferencesNudge.swift [hunk 3]
```diff
@@ -56,6 +56,7 @@ let mainContentSubHeader = getUserInterfaceUpdateElementsProfile()?["mainContent
 let mainContentText = getUserInterfaceUpdateElementsProfile()?["mainContentText"] as? String ?? userInterfaceUpdateElementsJSON?.mainContentText ?? "A fully up-to-date device is required to ensure that IT can accurately protect your device.\n\nIf you do not update your device, you may lose access to some items necessary for your day-to-day tasks.\n\nTo begin the update, simply click on the Update Device button and follow the provided steps.".localized(desiredLanguage: getDesiredLanguage())
 let primaryQuitButtonText = getUserInterfaceUpdateElementsProfile()?["primaryQuitButtonText"] as? String ?? userInterfaceUpdateElementsJSON?.primaryQuitButtonText ?? "Later".localized(desiredLanguage: getDesiredLanguage())
 let secondaryQuitButtonText = getUserInterfaceUpdateElementsProfile()?["secondaryQuitButtonText"] as? String ?? userInterfaceUpdateElementsJSON?.secondaryQuitButtonText ?? "I understand".localized(desiredLanguage: getDesiredLanguage())
+let singleQuitButton = userExperienceProfile?["singleQuitButton"] as? Bool ?? userExperienceJSON?.singleQuitButton ?? false
```
Comment (erikng) [inline Nudge/Preferences/DefaultPreferencesNudge.swift:59]: needs to move down now

**Review by erikng — CHANGES_REQUESTED** [review]

File: Nudge/Preferences/PreferencesStructure.swift [hunk 4]
```diff
@@ -361,6 +361,7 @@ struct UserInterface: Codable {
     var forceFallbackLanguage, forceScreenShotIcon: Bool?
     var iconDarkPath, iconLightPath, screenShotDarkPath, screenShotLightPath: String?
     var simpleMode: Bool?
+    var singleQuitButton: Bool?
```
Comment (erikng) [inline Nudge/Preferences/PreferencesStructure.swift:364]: nit, but in Xcode, if you have 2-4 variables in a struct like this with the same type, you want them on the same line. Please combine `simpleMode` and `singleQuitButton` together.

**Pull request merged**